crypto/bindings.ts

```diff
@@ -0,0 +1,9 @@
+import { prefixHashes, prefixHashesLegacy } from '../crypto/constants.js';
+
+const tsBindings = {
+  prefixHashes,
+  prefixHashesLegacy,
+};
+
+// this is put in a global variable so that ../kimchi/js/bindings.js finds it
+(globalThis as any).__snarkyTsBindings = tsBindings;
```

Comment:
this is the new TS bindings file, [explained here](https://github.com/o1-labs/snarkyjs/pull/1073#discussion_r1300137784)

Reply:
Nit: Depending how you feel on comments, you could argue the explanation could be added here, mostly just elaborating on the inline comment you have already. Could be helpful for others look at this file and wondering more, wdyt?